config.go

```diff
@@ -40,9 +41,18 @@ type Config struct {
 	// Tracer holds an optional apm.Tracer to use for tracing bulk requests
 	// to Elasticsearch. Each bulk request is traced as a transaction.
 	//
-	// If Tracer is nil, requests will not be traced.
+	// If Tracer is nil, requests will not be traced. Note however that
+	// OtelTracerProvider may not be nil, in which case the request will
+	// be traced by a different tracer.
```

Comment:
I'm tempted to add a deprecation notice to this, WDYT?

Reply:
I agree, done in [`a5fa1ad` (#182)](https://github.com/elastic/go-docappender/pull/182/commits/a5fa1ad7198cf9373558453db310f8588327a5b8)